shared/src/business/useCases/caseNote/getUserCaseNoteInteractor.test.js

```diff
@@ -42,7 +44,7 @@ describe('Get case note', () => {
   });
 
   it('throws an error if the entity returned from persistence is invalid', async () => {
-    applicationContext.getCurrentUser.mockReturnValue(mockJudge);
+    applicationContext.getCurrentUser.mockReturnValue(new User(mockJudge));
```

Comment:
We mock the current user for a bunch of these tests, but in the other tests for the user case notes (i.e. `getUserCaseNoteForCasesInteractor.js`) we only mock the new usecase helper.  So in these tests we're calling the real usecase helper I would assume.  Should we be consistent?  I think this also gets into the social versus unsocial unit tests.

Reply:
Oops, this comment doesn't make any sense anymore.

Reply:
I can't select the line because it is too far below the change in this PR, but line 51.  That's mocking the old interactor that isn't being called.  So we can change that to the usecase helper, right?